### DISCUSSION TITLE
Please explain how SESSION_EXPIRY_SECONDS works

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#13263] [Help] [answered]
Hi Budibase support,

I have tried setting SESSION_EXPIRY_SECONDS parameters at .env to 60 (1min) or 3600 (1hour) but I don't see session expiration (force login) for an idle user after this period. Budibase we have is self host 2.19.5 and according to my understanding this version should support this parameter. Can you explain how it works? Should we publish the app so that this change is available or is session timeout is controlled in backend? 
The goal is to set session timeout to 1 day instead of 1 week that is now documented is the default. 

Thank you.

**Comment by ConorWebb96** [comment]
Hi @gmentzik,

This functionality was only added 2 months ago. From past experience I believe you are on version `2.8.10`, you would need to be on version `2.15.7+` for this to work.

**Reply by ConorWebb96** [reply, marked as answer]
Below is some additional information,

1. Regarding SESSION_EXPIRY_SECONDS, it's essential to emphasise that the user must remain inactive for the duration specified.
2. If a user remains idle on the page, the system will not automatically log them out.
3. Additionally, accessing the page even once within the session expiry period will reset the timer.

Hopefully this makes sense.
